fedora/cinc/install_pkg.sh

```diff
@@ -114,6 +114,7 @@ yum -y --skip-broken install\
 	which\
 	fping\
 	perf\
+	git\
+	lksctp-tools-devel\
 	hostname
-
-
+``
```

Comment:
NIT: what is this last line doing (``)? Could it be it is not needed?
This is not really related to this change, so I'm just asking because this pr is changing this file. ;)

Reply:
Some how I added it by mistake. I'll delete it.

Reply:
Done